Mage/src/main/java/mage/abilities/effects/common/combat/CantAttackYouAllEffect.java

```diff
@@ -32,11 +32,11 @@ public CantAttackYouAllEffect(Duration duration, FilterCreaturePermanent filter,
         this.filterAttacker = filter;
         this.alsoPlaneswalker = alsoPlaneswalker;
         staticText = filterAttacker.getMessage() + " can't attack you"
-                + (alsoPlaneswalker ? " or a planeswalker you control" : "")
+                + (alsoPlaneswalker ? " or planeswalkers you control" : "")
```

Comment:
Confirming that the old rules has been errated out entirely:
[Does not exist anymore](https://scryfall.com/search?as=grid&extras=true&lang=any&order=name&q=o%3A%2Fcan%27t+attack+%5B%5E%2C%5D%2A+or+a+planeswalker+you+control%2F&unique=cards)
["New" wording](https://scryfall.com/search?q=o%3A%2Fcan%27t+attack+%5B%5E%2C%5D*+or+planeswalkers+you+control%2F&unique=cards&as=grid&order=name)

Reply:
Yeah I checked this and forgot to mention it. thanks for the reference.

Reply:
Those generated rules in shared effect definitively need more love. It's great to see them improve little by little.